Extension/src/Debugger/configurationProvider.ts

```diff
@@ -144,6 +144,7 @@ class CppConfigurationProvider implements vscode.DebugConfigurationProvider {
             newConfig.externalConsole = false;
             const exeName: string = path.join("${fileDirname}", "${fileBasenameNoExtension}");
             newConfig.program = platform === "win32" ? exeName + ".exe" : exeName;
+            newConfig.detail = definition.command;
```

Comment:
This just seems to duplicate a task's `command`. What if the user decides to change the `command` in their `tasks.json`, this `detail` will now be out of date and the user has to update this comment themselves. 

I would only just add this detail to the MenuItem and not actually return it to the user's `launch.json` configuration.

Reply:
@elahehrashedi Does just adding it to the MenuItem work?

Reply:
I removed the "detail" item after showing the menu Items. line #62

Reply:
Please add a comment saying this is not used in `launch.json` and it is just used to add a description for the MenuItem in `QuickPickConfigurationProvider.provideDebugConfigurations` and where `noDetailConfig.detail = undefined;` is set, add a comment saying this deletes the key so it does not get written to the `launch.json`